src/PrestaShopBundle/Resources/views/Admin/Module/Includes/tab-module-line.html.twig

```diff
@@ -24,7 +24,7 @@
  *#}
 {%
   set moduleImage = module.attributes.img is null ?
-    constant('_MODULE_DIR_') ~ module.attributes.name ~ '/' ~ module.attributes.logo :
+    public constant('_MODULE_DIR_') ~ module.attributes.name ~ '/' ~ module.attributes.logo :
```

Comment:
```suggestion
    constant('_MODULE_DIR_') ~ module.attributes.name ~ '/' ~ module.attributes.logo :
```

Reply:
Haha, I tried to read it all to find this kind of mistakes but I missed it!

Bravo 😉  very nice catch